protos/flyteidl/service/dataproxy.proto

```diff
@@ -63,6 +64,51 @@ message CreateDownloadLocationResponse {
   google.protobuf.Timestamp expires_at = 2;
 }
 
+enum ArtifactType {
+  ARTIFACT_TYPE_UNDEFINED = 0;
+  ARTIFACT_TYPE_DECK = 1;
+}
+
+message TaskExecutionId {
```

Comment:
Should we use [TaskExecutionIdentifier](https://github.com/flyteorg/flyteidl/blob/70aea35519c8849c26932567bdfbaf8f8fc55055/protos/flyteidl/core/identifier.proto#L59-L65) in `flyteidl.core` instead?

Reply:
oh yes! I didn't notice there is one. thank you!